_1327/static/less/1327.less

```diff
@@ -220,3 +220,11 @@ footer a {
 table.table-narrow {
 	max-width: 400px;
 }
+
+// Overwrite bootstrap table coloring
+.table-striped > tbody > tr:nth-of-type(odd) {
+	background-color: @white;
+}
+.table-striped > tbody > tr:nth-of-type(even) {
+	background-color: @gray-lighter;
+}
```

Comment:
Please introduce a new variable for this color (right next to ``@table-bg-accent``), put in a comment that it's not a bootstrap variable, and move this block to the bootstrap fixes at the top of the file.